docs/src/main/asciidoc/rest-client-reactive.adoc

```diff
@@ -394,7 +394,7 @@ public class ExtensionsResource {
     private final ExtensionsService extensionsService;
 
     public ExtensionsResource() {
-        extensionsService = RestClientBuilder.newBuilder()
+        extensionsService = QuarkusRestClientBuilder.newBuilder()
```

Comment:
Maybe we should add a call out explaining this?

Reply:
No no, I mean that in the docs we should explain the use of this new builder vs the Microprofile one.

Now users just see the class in snippets of code so might not realize what is going on.

Furthermore, experienced users will likely never see these snippets.

If instead we have a `TIP` or something, it will be a lot more visible

Reply:
+1 will update it

Reply:
PR updated.